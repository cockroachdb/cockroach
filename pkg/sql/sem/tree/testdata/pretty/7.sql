INSERT INTO mnop (m, n) SELECT i, (1e9 + i/2e4)::float FROM generate_series(1, 2e4) AS i(i) RETURNING NOTHING
