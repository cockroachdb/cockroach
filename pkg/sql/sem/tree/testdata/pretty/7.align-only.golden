// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
INSERT
INTO
	mnop
		(
			m,
			n
		)
SELECT
	i,
	(
		1e9
		+ i
			/ 2e4
	)::FLOAT8
FROM
	generate_series(
		1,
		2e4
	)
		AS i
			(
				i
			)
RETURNING
	NOTHING

9:
---------
   INSERT
     INTO mnop
			(
				m,
				n
			)
   SELECT i,
          (
			1e9
			+ i
				/ 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i
				(
					i
				)
RETURNING NOTHING

16:
----------------
   INSERT
     INTO mnop (
				m,
				n
               )
   SELECT i,
          (
			1e9
			+ i
				/ 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i
				(
					i
				)
RETURNING NOTHING

18:
------------------
   INSERT
     INTO mnop (
				m,
				n
               )
   SELECT i,
          (
			1e9
			+ i
				/ 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i (
					i
			     )
RETURNING NOTHING

20:
--------------------
   INSERT
     INTO mnop (
				m,
				n
               )
   SELECT i,
          (
			1e9
			+ i
				/ 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i (i)
RETURNING NOTHING

21:
---------------------
   INSERT
     INTO mnop (m, n)
   SELECT i,
          (
			1e9
			+ i / 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i (i)
RETURNING NOTHING

25:
-------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i,
          (
			1e9 + i / 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          )
			AS i (i)
RETURNING NOTHING

26:
--------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i,
          (
			1e9 + i / 2e4
          )::FLOAT8
     FROM generate_series(
			1,
			2e4
          ) AS i (i)
RETURNING NOTHING

33:
---------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i,
          (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(
			1,
			2e4
          ) AS i (i)
RETURNING NOTHING

36:
------------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i, (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(
			1,
			2e4
          ) AS i (i)
RETURNING NOTHING

38:
--------------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i, (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(1, 2e4) AS i
									(
										i
									)
RETURNING NOTHING

39:
---------------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i, (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(1, 2e4) AS i
									(i)
RETURNING NOTHING

40:
----------------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i, (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(1, 2e4) AS i (
										i
                                       )
RETURNING NOTHING

42:
------------------------------------------
   INSERT
     INTO mnop (m, n)
   SELECT i, (1e9 + i / 2e4)::FLOAT8
     FROM generate_series(1, 2e4) AS i (i)
RETURNING NOTHING

113:
-----------------------------------------------------------------------------------------------------------------
INSERT INTO mnop (m, n) SELECT i, (1e9 + i / 2e4)::FLOAT8 FROM generate_series(1, 2e4) AS i (i) RETURNING NOTHING


